docs/spec/v1beta2/providers.md

```diff
@@ -671,6 +672,55 @@ stringData:
   address: https://chat.googleapis.com/v1/spaces/...
 ```
 
+##### Google Pub/Sub
+
+When `.spec.type` is set to `googlepubsub`, the controller will publish the payload of
+an [Event](events.md#event-structure) on the Google Pub/Sub Topic provided in the
+[Address](#address) field of the Provider (the format must be
+`projects/<project ID>/topics/<topic ID>`).
```

Comment:
Suggestion: For [Azure Event Hub](https://fluxcd.io/flux/components/notification/provider/#azure-event-hub) we're using `address` for the hub name and `channel` for the namespace. Why not do this similarly here as well?

```
kind: Provider
[...]
spec:
  address: <project ID>
  topic: <topic ID>
```